README.md

```diff
@@ -0,0 +1,14 @@
+![](https://github.com/openmainframeproject/artwork/raw/master/projects/softwarediscoverytool/softwarediscoverytool-color.svg)
+
+# Data for Software Discovery Tool
+  
+![License](https://img.shields.io/github/license/openmainframeproject/software-discovery-tool)
```

Comment:
I don't expect the license to differ between these projects, but we should still use the image specifically generated for this one, so the one you want is:

https://img.shields.io/github/license/openmainframeproject/software-discovery-tool-data